### ISSUE TITLE
Does not work with disabled outputs

### ISSUE THREAD

**Issue by farion** [moverest/sway-interactive-screenshot#13]
Hi,

I have disabled some outputs (via kanshi), which leads to this error:

```
Traceback (most recent call last):
  File "/usr/local/bin/sway-interactive-screenshot", line 819, in <module>
    main()
  File "/usr/local/bin/sway-interactive-screenshot", line 757, in main
    choices = [
  File "/usr/local/bin/sway-interactive-screenshot", line 343, in get_outputs
    focused=output["focused"],
KeyError: 'focused'
```

**Comment by farion** [comment]
Seems to be fixed already.

**Comment by moverest** [comment]
Indeed, this issue was fixed with https://github.com/moverest/sway-interactive-screenshot/pull/11 and released with [2.0.1](https://github.com/moverest/sway-interactive-screenshot/releases/tag/2.0.1).
